src/patternfly/components/Content/content.scss

```diff
@@ -109,6 +110,13 @@
     }
   }
 
+  &.pf-m-visited,
+  .pf-m-visited {
+    &:visited {
+      color: var(--pf-c-content--a--visited--Color);
+    }
+  }
+
```

Comment:
This compiles to

```
.pf-c-content.pf-m-visited:visited,
.pf-c-content .pf-m-visited:visited {
  color: var(--pf-c-content--a--visited--Color);
}
```

Which won't target all `:visited` links in a content block with `.pf-m-visited`

I think maybe you meant to do this?

```
.pf-c-content {
  &.pf-m-visited :visited,
  .pf-m-visited:visited {
      color: var(--pf-c-content--a--visited--Color);
  }
}
```

Which compiles to:

```
.pf-c-content.pf-m-visited :visited, /* targets links in a block with .pf-m-visited */
.pf-c-content .pf-m-visited:visited { /* targets a link with .pf-m-visited */
  color: var(--pf-c-content--a--visited--Color);
}
```

Another way would be to group the styles with the `a` element above, it seems more appropriate there:

```
    @at-root .pf-c-content.pf-m-visited a,
    &.pf-m-visited {
        &:visited {
            --pf-c-content--a--Color: var(--pf-c-content--a--visited--Color);
        }
    }
```

which compiles to:

```
.pf-c-content.pf-m-visited a:visited,
.pf-c-content a.pf-m-visited:visited {
  --pf-c-content--a--Color: var(--pf-c-content--a--visited--Color);
}
```

@mcarrano @mceledonia can you remind me if the eventual intent is to have all text links (that aren't part of a component, like nav, tabs, dropdown items, etc) have visited styles, and we'll make that update in a breaking change release? If so, we should probably also add the ability to make a single outside of the content component have visited styles by adding that modifier to the global `a` rules in https://github.com/patternfly/patternfly/blob/78eb519250b9aaf53391f366688f92f1d420c0ed/src/patternfly/base/_globals.scss#L114-L123

Reply:
@mcoker I don't know under what circumstances you would want to apply visited styles in places other than the content component, but it probably wouldn't hurt to allow that modifier to be globally available just in case someone wants to use it.

Reply:
@mcarrano it makes it a lot easier on our end if we limit these styles just to the content component. Then we don't have to worry about the overlapping styles that will result from global `:visited` styles interfering with component link elements that shouldn't have a `:visited` state - like nav, tabs, etc. If the eventual intent is to apply visited styles to links globally though, it's probably good to go ahead and introduce this variation for global links.

Reply:
I'm good to keep this localized to the content component for now.